roles/variables/defaults/main.yml

```diff
@@ -330,7 +330,7 @@ zookeeper_jolokia_port: 7770
 zookeeper_jolokia_ssl_enabled: "{{ zookeeper_ssl_enabled }}"
 
 ### Path on Zookeeper host for Jolokia Configuration file
-zookeeper_jolokia_config: "{{ (config_base_path, 'etc/kafka/zookeeper_jolokia.properties' ) | community.general.path_join }}"
+zookeeper_jolokia_config: "{{ ((config_base_path, zookeeper_config_prefix) | join, 'zookeeper_jolokia.properties') | community.general.path_join }}"
```

Comment:
path_join joins multiple paths together. Please see if you really need to use `join`

Reply:
thanks Neeraj, as discussed, have made another commit implementing cleaner way of joining paths, PTAL